pom.xml

```diff
@@ -100,6 +100,7 @@
                 <configuration>
                     <configFile>${project.basedir}/formatter.xml</configFile>
                     <encoding>UTF-8</encoding>
+                    <lineEnding>KEEP</lineEnding>
```

Comment:
I added this here because formatting was changing line endings on my machine.

Reply:
Maybe forcing a style would be a good idea, it looks like there's a mix inside the project.
See the documentation [here](https://code.revelc.net/formatter-maven-plugin/format-mojo.html#lineEnding)

Reply:
we have an open PR for that :)